enhancements/machine-api/out-of-tree-provider-support.md

```diff
@@ -198,18 +198,39 @@ With the `--external-cloud-volume-plugin` flag set, existing designs for this tr
 
 #### Pre-release / Development Feature Gate
 
-In order to achieve desired cluster configuration for cloud components during development and technical preview, we will add a new [feature gate](https://docs.openshift.com/container-platform/4.5/nodes/clusters/nodes-cluster-enabling-features.html) `ExternalCloudProvider`, which will allow developers and customers to force their cluster to upgrade from in-tree to out-of-tree cloud providers.
+In order to achieve desired cluster configuration for cloud components during development and technical preview, we will add a new OpenShift [FeatureGate](https://docs.openshift.com/container-platform/4.5/nodes/clusters/nodes-cluster-enabling-features.html) resource named `ExternalCloudProvider`, which will allow developers and customers to force their cluster to upgrade from in-tree to out-of-tree cloud providers.
 
-By using this feature gate with `TechPreviewNoUpgrade` `FeatureSet`, users will prevent their clusters from upgrading to newer versions of OpenShift as with all other feature gates.
+This feature gate will be shipped with `TechPreviewNoUpgrade` `FeatureSet`, preventing upgrades.
 
 The feature gate will be used to determine whether or not operators should migrate their components to the out-of-tree configuration.
-If the feature gate  is listed (via a custom or technical preview feature set), the operators for components such as `kube-apiserver`, `kube-controller-manager` and `kubelet` will update their component's configuration accordingly to allow hosting external `CCM` for any platform.
+If the feature gate is found, the operators for components such as `kube-apiserver`, `kube-controller-manager` and `kubelet` will update their component's configuration accordingly to allow hosting external `CCM` for current platform.
 
-Once an out-of-tree provider implementation is ready to be released (GA), it will no longer depend on the `TechPreviewNoUpgrade` feature gate and will automatically upgrade as part of a 4.N to 4.N+1 upgrade using default feature set.
+Once an out-of-tree provider implementation is ready to be released (GA), it will no longer depend on the feature gate, allowing production clusters to automatically upgrade to the out-of-tree implementation during 4.N to 4.N+1 upgrade.
 
 Once all providers are moved out-of-tree, the feature gate will be safe to remove.
 
-This feature gate will be commonly used in CI tasks, to ensure upgrade stability.
+This feature gate will be used in CI tasks, to test migration stability.
+
```

Comment:
Describe what happens when the gate is unset again.

Reply:
General question, what happens with other feature gates when they are unset? Is it expected that they disable the feature, or, given these are experimental and block upgrades, are we supposed to ignore it being unset?

Reply:
Disable the feature is the answer. Covered in the proposal.